### PR TITLE
simple-adblock: add compressed blocklist store option, proper filter …

### PR DESCRIPTION
**Pull request by stangri**
…of non-ASCII domains

Signed-off-by: Stan Grishin <stangri@melmac.net>

Maintainer: me
Compile tested: mvebu, WRT3200, OpenWrt 18.06.2
Run tested: mvebu, WRT3200, OpenWrt 18.06.2

Please backport to the current OpenWrt packages, as it fixes an issue where dnsmasq may not start.

### CODE REVIEW

**Comment by neheb** [comment]
You can open another pull request. Just change the branch that it should go in.

**Comment by stangri** [comment]
> You can open another pull request. Just change the branch that it should go in.

As my branch is based on master, should I ignore the github's warning "Can’t automatically merge."?

If not, could you please help me with git-foo on how to create new PRs for openwrt-18.06 and openwrt-19.07 which could be automatically merged?

Thanks!

**Comment by neheb** [comment]
I usually do

```
git checkout openwrt-19.07
git checkout -b branch
Make changes
git add .
git commit --signoff
git push repo
```

Then make a pull request to the original branch.

File: net/simple-adblock/Makefile [hunk 1]
```diff
@@ -5,8 +5,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=simple-adblock
-PKG_VERSION:=1.6.3
-PKG_RELEASE:=5
+PKG_VERSION:=1.7.0
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-3.0+
```
Comment (BKPepe) [inline net/simple-adblock/Makefile]:
```suggestion

PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
PKG_LICENSE:=GPL-3.0-or-later
```

File: net/simple-adblock/Makefile [hunk 2]
```diff
@@ -7,7 +7,9 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=simple-adblock
 PKG_VERSION:=1.7.0
 PKG_RELEASE:=1
-PKG_LICENSE:=GPL-3.0+
+
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
+PKG_LICENSE:=GPL-3.0-or-later
 PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
```
Comment (BKPepe) [inline net/simple-adblock/Makefile]: There is now duplicated row. Remove the PKG_MAINTAINER, which is under PKG_LICENSE.

File: net/simple-adblock/files/README.md [hunk 3]
```diff
@@ -18,113 +20,120 @@ A simple DNSMASQ-based AdBlocking service for OpenWrt/LEDE Project. Loosely base
...
 
 If you specify ```google.com``` as a domain to be whitelisted, you will have access to ```google.com```, ```www.google.com```, ```analytics.google.com```, but not fake domains like ```email-google.com``` or ```drive.google.com.verify.signin.normandeassociation.com``` for example. If you only want to allow ```www.google.com``` while blocking all other ```google.com``` subdomains, just specify ```www.google.com``` as domain to be whitelisted.
 
 In general, whatever domain is specified to be whitelisted; it, along with with its subdomains will be whitelisted, but not any fake domains containing it.
 
 ## Documentation / Discussion
+
 Please head [LEDE Project Forum](https://forum.lede-project.org/t/simple-adblock-fast-lean-and-fully-uci-luci-configurable-adblocking/1327/) for discussion of this package.
```
Comment (BKPepe) [inline net/simple-adblock/files/README.md]:
```suggestion
Please head [OpenWrt forum](https://forum.openwrt.org/t/simple-adblock-fast-lean-and-fully-uci-luci-configurable-adblocking/1327) for discussion of this package.
```

**Comment by stangri** [comment]
I'll resend the PR for different branches tomorrow.

**Pull request closed without merging**